v22.1/restore.md

```diff
@@ -232,32 +243,41 @@ The examples in this section use the **default** `AUTH=specified` parameter. For
 (3 rows)
 ~~~
 
-When you restore a backup, add the backup's subdirectory path (e.g. `/2021/12/21-142943.73`) to the `RESTORE` statement.
+When you want to [restore a specific backup](#restore-a-specific-backup), add the backup's subdirectory path (e.g., `/2021/12/21-142943.73`) to the `RESTORE` statement. For details on viewing the most recent backup, see [`SHOW BACKUP LATEST`](show-backup.html#show-the-most-recent-backup).
 
-Incremental backups will be appended to the full backup with `BACKUP ... INTO LATEST IN {destination}`. Your storage location will contain the incremental as a date-based subdirectory within the full backup.
+Incremental backups will be appended to the full backup with `BACKUP ... INTO LATEST IN {collection location}`. Your storage location will contain the incremental in the default `/incrementals` directory at the collection location's root. For more detail on collections, see [Backup Collections](take-full-and-incremental-backups.html#backup-collections).
 
-In the following example `/2021/12/21-142943.73` contains the full backup. The incremental backups (`144748.08/` and `144639.97/`) are appended as subdirectories to the full backup:
+To output more detail about the backups contained within a directory, see [View a list of the full and incremental backups in a specific full backup subdirectory](show-backup.html#view-a-list-of-the-full-and-incremental-backups-in-a-specific-full-backup-subdirectory).
```

Comment:
ditto: im not sure this sentence is necessary

Reply:
Yes I agree, in general I think that section had got overworked over the release... will remove the last three sentences.